modules/placement-driver-api/src/testFixtures/java/org/apache/ignite/internal/placementdriver/TestPlacementDriver.java

```diff
@@ -69,6 +69,6 @@ public void removeListener(PrimaryReplicaEvent evt, EventListener<? extends Prim
 
     @Override
     public CompletableFuture<Void> previousPrimaryExpired(ReplicationGroupId grpId) {
-        return CompletableFuture.completedFuture(null);
+        return completedFuture(null);
```

Comment:
Interesting. Import is not updated. Why did you do it? :)

Reply:
For beauty.